src/components/link-detail/index.js

```diff
@@ -275,6 +275,11 @@ export class LinkDetail extends React.Component {
                 placeholder="username"
                 value={this.state.upstreamOwner}
                 onChange={e => {
+                  // Remove the `github.com`, if the user pasted a full GitHub link
+                  var githubMatchExpression = new RegExp("https?:\\/\\/github\\.com\\/", "")
```

Comment:
Can you use `let` instead of `var`, also use a literal regex instead of the constructor?